__tests__/bin/vip-logs.js

```diff
@@ -28,6 +28,7 @@ jest.mock( 'lib/tracker', () => ( {
 } ) );
 
 jest.mock( 'lib/app-logs/app-logs', () => ( {
+	...jest.requireActual( 'lib/app-logs/app-logs' ),
```

Comment:
Can we add in inline comment noting why we're doing this? I'm guessing this is because we just want to mock `getRecentLogs` but it's not super obvious.

Reply:
I added this because the constant `LIMIT_MAX` defined in the module would end up being `undefined` during the test execution. 
Comment added in 6ae2a73ce4f93b8111537a2344adfcaaecc8b57b